docker-compose.yml

```diff
@@ -59,26 +59,23 @@ services:
     # Using a volume allows autorebuild to work.
     volumes:
       - ./:/usr/src/app
-    # Map the default port.
+    # List ports the container should expose
+    # For production you might want to add "80:3000" in place of the "3000:3000" line below
     ports:
-      - "3000:3000" # Should be commented out if you uncomment 80:3000 below.
-      - "9229:9229" # Debug port, should be commented out for production
-      # For production you might want something like:
-      # - "80:3000"
-      # and comment out the debug port and 3000:3000 line above
-      # Don't bring this up without the DB
+      - "3000:3000"
+      - "9229:9229" # Development port; should be commented out for production
+      - "9230:9230"
```

Comment:
This is one of the issues that I'll mention in my update comment: In Docker, Vite does not listen on port `9229` because it is "already in use" - so as a fallback it listens on the port above that - `9230`. As a workaround, expose port "9230" so things work.

Reply:
This issue has been fixed in eb0f7091c3ec53dacbe243967d94fc579e7d1d4b